src/nowsecure-action.ts

```diff
@@ -78,4 +97,31 @@ async function run() {
   }
 }
 
+export async function outputToDependencies(
+  report: PullReportResponse,
+  context: ActionContext,
+  githubToken: string
+) {
+  if (report?.data?.auto?.assessments[0]?.deputy) {
+    console.log("Converting NowSecure report to Snapshot format...");
+    const deputy = report?.data?.auto?.assessments[0]?.deputy;
```

Comment:
This could be moved up to avoid repeating this long chain of checks, simplifying the `if ()` above.